__tests__/app.test.js

```diff
@@ -116,3 +116,44 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an article object with the article_id from the URL", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments }  }) => {
+        expect(Array.isArray(comments)).toBe(true);
+        expect(comments.length).not.toBe(0);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
any `article id`?

Reply:
I've changed it in another pull request. Here is the link https://github.com/ghiffari90/my-nc-news/pull/10 :)